### ISSUE TITLE
[Issue]: Logos are not available 

### ISSUE THREAD

**Issue by Sledgehamma** [jellyfin/jellyfin#6942]
### Please describe your bug

Setup:
Running Jellyfin 10.7.7 on unRaid 6.9.2. via Docker (ich777)

Movies:
Metadata downloaders:
1. TheMovieDB
2. The Open Movie Database

Fetcher Settings:
1. TheMovieDB
2. The Open Movie Database
3. Screen Grabber

TV Shows:
Metadata downloaders:
1. TheMovieDB
2. The Open Movie Database

Episode Metadata downloaders:
1. TheMovieDB
2. The Open Movie Database

Series Fetcher Settings:
TheMovieDB

Season Fetcher Settings:
TheMovieDB

Issue:
Logos are not downloaded for either movies nor tv shows. Even manually going to a movie then "edit images"  and going straight to "Logos" none are shown. Also enabling "all languages" doesn't load any logos.

See one example:

Spiderman: Far From Home (2019):
Logos are available on TMDB
https://www.themoviedb.org/movie/429617-spider-man-far-from-home/images/logos

However, none are shown in Jellyfin (please see attached screenshot)
Same behavior for TV shows. I couldn't find any example in which logos are available. 

Please let me know if I should provide additional information. Thanks for your hard work!

Cheers!

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Please attach any screenshots here

<img width="1364" alt="Screenshot 2021-12-05 at 09 27 24" src="https://user-images.githubusercontent.com/19309778/144739386-12074a6a-b081-4e14-9d65-cf78cf7931a0.png">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by 1337joe** [comment]
Logos are not available from TMDb because the library we use doesn't yet have a release to support them. Support for logos is merged into TMDbLib (LordMike/TMDbLib#389). As soon as that's released and we're updated to use the new version I can merge [this](https://github.com/1337joe/jellyfin/commit/7edfcb0ba853ff951eabb36ad790a5d134ee7109) to add support to jellyfin.

**Comment by Sledgehamma** [comment]
Awesome! Thanks for your answer and the link :)
